### PR TITLE
fix: correct the ExampleModal file

### PR DESCRIPTION
**Pull request by Azn9**
**Description:** Update the ExampleModal file

**Justification:** The example previously provided was using SelectMenu in modals, which are not allowed. For now only TextInput are allowed in modals.

### CODE REVIEW

**Comment by Doc94** [comment]
This not apply for 3.2.x?

File: core/src/test/java/discord4j/core/ExampleModal.java [hunk 1]
```diff
@@ -59,35 +57,29 @@ public static void main(String[] args) {
...
-                                                    SelectMenu.Option.of("one", "1"),
-                                                    SelectMenu.Option.of("two", "2"),
-                                                    SelectMenu.Option.of("three", "3"))
-                                            .withMinValues(0)))
-                                            .addComponent(ActionRow.of(TextInput.small(
-                                                    INPUT_CUSTOM_ID, "Something else to add?")
-                                                    .required(false)))
+                                    .addAllComponents(Arrays.asList(
+                                            ActionRow.of(TextInput.small(PARAGRAPHINPUT_CUSTOM_ID, "Tell us something...", 250, 928).placeholder("...in more than 250 characters but less than 928").required(true)),
+                                            ActionRow.of(TextInput.small(INPUT_CUSTOM_ID, "Something to add?").required(false))
```
Comment (Azn9) [inline core/src/test/java/discord4j/core/ExampleModal.java:62]:
```suggestion
                                            ActionRow.of(TextInput.small(INPUT_CUSTOM_ID, "A title?").required(false))
                                            ActionRow.of(TextInput.paragraph(PARAGRAPHINPUT_CUSTOM_ID, "Tell us something...", 250, 928).placeholder("...in more than 250 characters but less than 928").required(true))
```

File: core/src/test/java/discord4j/core/ExampleModal.java [hunk 2]
```diff
@@ -59,35 +57,29 @@ public static void main(String[] args) {
...
 
                     Publisher<?> onModal = client.on(ModalSubmitInteractionEvent.class, event -> {
                         if (MODAL_CUSTOM_ID.equals(event.getCustomId())) {
+                            String story = "";
                             String comments = "";
+
                             for (TextInput component : event.getComponents(TextInput.class)) {
-                                if (INPUT_CUSTOM_ID.equals(component.getCustomId())) {
+                                if (PARAGRAPHINPUT_CUSTOM_ID.equals(component.getCustomId())) {
+                                    story = component.getValue().orElse("");
```
Comment (Azn9) [inline core/src/test/java/discord4j/core/ExampleModal.java:76]:
```suggestion
                                    story = component.getValue().orElse("untiteled");
```

File: core/src/test/java/discord4j/core/ExampleModal.java [hunk 3]
```diff
@@ -59,35 +57,29 @@ public static void main(String[] args) {
...
-                                    .addComponent(ActionRow.of(SelectMenu.of(SELECT_CUSTOM_ID,
-                                                    SelectMenu.Option.of("one", "1"),
-                                                    SelectMenu.Option.of("two", "2"),
-                                                    SelectMenu.Option.of("three", "3"))
-                                            .withMinValues(0)))
-                                            .addComponent(ActionRow.of(TextInput.small(
-                                                    INPUT_CUSTOM_ID, "Something else to add?")
-                                                    .required(false)))
+                                    .addAllComponents(Arrays.asList(
+                                            ActionRow.of(TextInput.small(INPUT_CUSTOM_ID, "A title?").required(false))
```
Comment (Azn9) [inline core/src/test/java/discord4j/core/ExampleModal.java:61]:
```suggestion
                                            ActionRow.of(TextInput.small(INPUT_CUSTOM_ID, "A title?").required(false)),
```

**Review by Doc94 — APPROVED** [review]

**Pull request merged**